lib/ansible/modules/cloud/azure/azure_rm_sql_database.py

```diff
@@ -0,0 +1,442 @@
+               ties_ListByLocation.'"
+        required: False
+    requested_service_objective_id:
+        description:
+            - "The configured service level objective ID of the database. This is the service level objective that is in the process of being applied to the
+               database. Once successfully updated, it will match the value of currentServiceObjectiveId property. If requestedServiceObjectiveId and request
+               edServiceObjectiveName are both updated, the value of requestedServiceObjectiveId overrides the value of requestedServiceObjectiveName. To see
+                possible values, query the capabilities API (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) ref
+               erred to by operationId: 'Capabilities_ListByLocation.'"
+        required: False
+    requested_service_objective_name:
+        description:
+            - "The name of the configured service level objective of the database. This is the service level objective that is in the process of being applie
+               d to the database. Once successfully updated, it will match the value of serviceLevelObjective property. To see possible values, query the cap
+               abilities API (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) referred to by operationId: 'Capab
+               ilities_ListByLocation.'. Possible values include: 'Basic', 'S0', 'S1', 'S2', 'S3', 'P1', 'P2', 'P3', 'P4', 'P6', 'P11', 'P15', 'System', 'Sys
+               tem2', 'ElasticPool'"
+        required: False
+    elastic_pool_name:
+        description:
+            - "The name of the elastic pool the database is in. If elasticPoolName and requestedServiceObjectiveName are both updated, the value of requested
+               ServiceObjectiveName is ignored. Not supported for DataWarehouse edition."
+        required: False
+    read_scale:
+        description:
+            - "Conditional. If the database is a geo-secondary, readScale indicates whether read-only connections are allowed to this database or not. Not su
+               pported for DataWarehouse edition. Possible values include: 'Enabled', 'Disabled'"
+        required: False
+    sample_name:
+        description:
+            - "Indicates the name of the sample schema to apply when creating this database. If createMode is not Default, this value is ignored. Not support
+               ed for DataWarehouse edition. Possible values include: 'AdventureWorksLT'"
+        required: False
+    zone_redundant:
+        description:
+            - Whether or not this database is zone redundant, which means the replicas of this database will be spread across multiple availability zones.
+        required: False
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+      - name: Create (or update) Sql
```

Comment:
A few things

* Keep the examples left aligned (remove the leading 4 spaces)
* Stop using jinja2 block `{{}}` in the examples so people can see what the actual may look like
* Provide more than 1 example, usually we have a basic with only the parameters required, then 1 for each major scenario, like 1 for the main create_mode options and so on